samples/word/50-document/manage-settings.yaml

```diff
@@ -0,0 +1,133 @@
+    content: |
+        $("#create-setting").click(() => tryCatch(createSetting));
+        $("#get-settings").click(() => tryCatch(getSettings));
+        $("#delete-all-settings").click(() => tryCatch(deleteAllSettings));
+
+        async function createSetting() {
+          await Word.run(async (context) => {
+            let key = $("#key")
+              .val()
+              .toString();
+            let value = $("#value")
+              .val()
+              .toString();
+            const settings = context.document.settings;
+            let setting = settings.add(key, value);
+            setting.load();
+            await context.sync();
+
+            console.log("Setting added:");
+            console.log(setting);
+          });
+        }
+
+        async function getSettings() {
+          await Word.run(async (context) => {
+            let settings = context.document.settings;
+            settings.load("items");
+            await context.sync();
+
+            const allSettings = [];
+            for (let i = 0; i < settings.items.length; i++) {
+              let setting = settings.items[i];
+              setting.load();
+              allSettings.push(setting);
+            }
+
+            await context.sync();
+
+            if (allSettings.length <= 0) {
+              console.log("There are no settings");
+            } else {
+              console.log("All settings:");
+              for (let i = 0; i < allSettings.length; i++) {
+                console.log(allSettings[i]);
+              }
+            }
+          });
+        }
+
+        async function deleteAllSettings() {
+          await Word.run(async (context) => {
+            let settings = context.document.settings;
+            settings.deleteAll();
+            await context.sync();
+            console.log("All settings deleted");
+          });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+          try {
+            await callback();
+          } catch (error) {
+            // Note: In a production add-in, you'd want to notify the user through your add-in's UI.
+            console.error(error);
+          }
+        }
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to add, get, and delete settings in the file.</p>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <p>Add a setting</p>
+            <p>
+                <div class="ms-TextField">
+                    <label class="msLabel">Key:</label>
+                    <input id="key" class="msTextField-field" type="text">
+                </div>
+                    <div class="ms-TextField">
+                        <label class="msLabel">Value:</label>
+                        <input id="value" class="msTextField-field" type="text">
+                </div>
```

Comment:
Small nit, but is it possible to get these text input fields to line up?

![image](https://user-images.githubusercontent.com/38896772/194654427-9423ee28-3d00-4c38-ae50-a165ee1256f4.png)